compiler/frontend/src/org/jetbrains/kotlin/checkers/CheckerTestUtil.java

```diff
@@ -583,11 +583,11 @@ public SyntaxErrorDiagnostic(@NotNull PsiErrorElement errorElement) {
     }
 
     public static class DebugInfoDiagnosticFactory extends DiagnosticFactory<DebugInfoDiagnostic> {
-        public static final DebugInfoDiagnosticFactory SMARTCAST = new DebugInfoDiagnosticFactory("SMARTCAST");
+        public static final DebugInfoDiagnosticFactory SMARTCAST = new DebugInfoDiagnosticFactory("SMARTCAST", Severity.INFO);
```

Comment:
We've discussed internally this change: it's OK, but could you please also change `Severity` to `INFO` for: `IMPLICIT_RECEIVER_SMARTCAST`,  `CONSTANT` and `DYNAMIC`